docs/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'PMDA'
-author = u'Max Linke, Mahzad Khoslessan, Richard J Gowers, Oliver Beckstein'
+author = u'Max Linke, Mahzad Khoshlessan, Richard J Gowers, Oliver Beckstein'
```

Comment:
@mkhoshle : the change fixed your name to read "Khoshlessan" (look at the "Files changed" to see the diff).

Is this ok now?